src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -115,8 +132,7 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     assert( newSubmission.workflows.size == 3 )
   }
 
-  //TODO: re-enable when empty sample sets are serialized out of Orient correctly
-  /*"Submission requests"*/ ignore should "return a successful Submission when given an entity expression that evaluates to an empty set of entites" in withWorkspaceService { workspaceService =>
+  "Submission requests" should "return a successful Submission when given an entity expression that evaluates to an empty set of entites" in withWorkspaceService { workspaceService =>
```

Comment:
I re-enabled this test (it was ignored) since we can serialize empty sets now :)